Tests/ApolloServerIntegrationTests/SchemaRegistryApolloSchemaDownloaderTests.swift

```diff
@@ -0,0 +1,31 @@
+#if os(macOS)
+import XCTest
+import ApolloTestSupport
+import ApolloCodegenTestSupport
+@testable import ApolloCodegenLib
+
+class SchemaRegistryApolloSchemaDownloaderTests: XCTestCase {
+  func testDownloadingSchema_fromSchemaRegistry_shouldOutputSDL() throws {
+    let testOutputFolderURL = CodegenTestHelper.outputFolderURL()
+    XCTAssertFalse(FileManager.default.apollo.fileExists(at: testOutputFolderURL))
+
+    guard let apiKey = ProcessInfo.processInfo.environment["REGISTRY_API_KEY"] else {
```

Comment:
What's the difference between this and the other test that uses `REGISTRY_API_KEY`, additional specificity?

Reply:
I moved the two schema downloading tests out of `ApolloSchemaTests.swift` and the `ApolloCodegenTests` target because:
* The starwars introspection download test would have required the test to spin up the starwars server. Since that was already happening in this target (`ApolloServerIntegrationTests`) I felt it was better suited to live here.
* That made me bring across the registry-based schema download test to group tests that rely on external services - no other reason. So the 'other' registry download test isn't there anymore.
* I added [a schema-based test that uses static JSON](https://github.com/apollographql/apollo-ios/pull/1935/commits/75b72785c477995a3020a3cf9e44c5e6811ac336#diff-76997a5d08e3f423718fcbb13c8baaf45ad038ac8938936781856373b3df5db3R66) instead so it didn't need to actually download anything.

Reply:
ah right that all got moved while I was working on all this. thank you!